worlds/AutoWorld.py

```diff
@@ -22,9 +21,9 @@
 
 
 class AutoWorldRegister(type):
-    world_types: Dict[str, Type[World]] = {}
+    world_types: dict[str, Type[World]] = {}
```

Comment:
```suggestion
    world_types: dict[str, type[World]] = {}

```
also adding a blank line to separate `ClassVar`s from non-`ClassVar`s